src/main/kotlin/no/nav/syfo/personoppgavehendelse/PersonoppgavehendelseProducer.kt

```diff
@@ -33,7 +33,9 @@ class PersonoppgavehendelseProducer(
                 personoppgaveId.toString(),
                 kPersonoppgavehendelse,
             )
-            producer.send(record).get()
+            producer.send(record).also {
```

Comment:
Dette er den eneste kodeendringen jeg har måttet gjøre for å få testene til å kjøre.